### DISCUSSION TITLE
Configure Navigation to display only items in the current section?

### DISCUSSION THREAD

**Discussion by Mrt134** [thegeeklab/hugo-geekdoc#613] [Q&A]
I've got a pretty heavy site that has multiple sections under the main content folder:
```
/content
  /section1
  /section2
  /section3
  /section4
  /section5
```
I'm curious if I can reset the navigation sidebar to be invisible on the home page (already done) so I can create a custom menu with links to each major section and then, when visiting a section home page, the Navigation sidebar only shows the content files contained within that section?
I've done some digging and have seen another hugo theme (https://unicef.github.io/inventory/) do roughly what I'm looking for, but I haven't been able to successfully adapt that method to the existing Geekdoc theme. Any advice?

**Comment by xoxys** [comment]
That's not (easily) possible with the theme. It would also need some work to build a context agnostic main menu. If someone wants to provide a Pull Request, I'm happy to review, but currently I can't work another menu system on my own.
